packages/examples/src/billing/insurance-eligibility-checks.ts

```diff
@@ -0,0 +1,131 @@
+import { CoverageEligibilityRequest, CoverageEligibilityResponse } from '@medplum/fhirtypes';
+
+const eligibilityCheck: CoverageEligibilityRequest =
+  // start-block eligibilityRequest
+  {
+    resourceType: 'CoverageEligibilityRequest',
+    id: 'coverage-validation-request',
+    status: 'active',
+    purpose: ['validation'],
+    patient: {
+      reference: 'Patient/homer-simpson',
+    },
+    provider: {
+      reference: 'Practitioner/dr-alice-smith',
+    },
+    insurer: {
+      reference: 'Organization/blue-cross-blue-shield',
+    },
+    insurance: [
+      {
+        coverage: {
+          reference: 'Coverage/example-coverage',
+        },
+      },
+    ],
+    item: [
+      {
+        category: {
+          coding: [
+            {
+              system: 'http://terminology.hl7.org/CodeSystem/ex-benefitcategory',
+              code: '3',
+              display: 'Consultation',
+            },
+          ],
+        },
+        productOrService: {
```

Comment:
Include X12? Candid, Change and others use X12 service codes https://developers.changehealthcare.com/eligibilityandclaims/docs/use-bare-minimum-eligibility-requests